### PR TITLE
Fix typos and grammar issues in README

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Signed-off-by: Lucian <lucian.buzzo@gmail.com>

### CODE REVIEW

**Comment by domenic** [comment]
Thanks, but I disagree with all your comma choices. The correction of "loads" to "load", and the cross-link addition are welcome, but on balance this patch makes the readme harder to read, so I'd rather not take it.

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ console.log(dom.window.document.querySelector("p").textContent); // "Hello world
 
 (Note that jsdom will parse the HTML you pass it just like a browser does, including implied `<html>`, `<head>`, and `<body>` tags.)
 
-The resulting object is an instance of the `JSDOM` class, which contains a number of useful properties and methods besides `window`. In general it can be used to act on the jsdom from the "outside," doing things that are not possible with the normal DOM APIs. For simple cases, where you don't need any of this functionality, we recommend a coding pattern like
+The resulting object is an instance of the `JSDOM` class, which contains a number of useful properties and methods besides `window`. In general, it can be used to act on the jsdom from the "outside," doing things that are not possible with the normal DOM APIs. For simple cases, where you don't need any of this functionality, we recommend a coding pattern like
```
Comment (LucianBuzzo) [inline README.md]: The use of a comma here is consistent with how the phrase is used elsewhere in the README:
- https://github.com/jsdom/jsdom/blame/master/README.md#L174
- https://github.com/jsdom/jsdom/blame/master/README.md#L6

Comment (LucianBuzzo) [reply README.md]: I've used blame mode as I can't figure out how to render line numbers for `.md` files otherwise!

**Comment by LucianBuzzo** [comment]
@domenic Thanks for the feedback. I've updated my PR to remove the comma changes, with the exception of one ( `In general,` ) which has a precedent elsewhere in the text.

**Comment by LucianBuzzo** [comment]
@domenic Could you take another look at this PR please?

**Comment by domenic** [comment]
Thanks!

**Pull request merged**